terraform/k8s_templates/middle-layer-fleet.yaml.tpl

```diff
@@ -0,0 +1,54 @@
+apiVersion: agones.dev/v1
+kind: Fleet
+metadata:
+  name: middle-layer-app
+spec:
+  replicas: 10
+  template:
+    spec:
+      health:
+        initialDelaySeconds: 20
+        periodSeconds: 10
+      ports:
+        - name: default
+          containerPort: 7101
+          protocol: TCP
+      template:
+        spec:
+          affinity:
+            nodeAffinity:
+              requiredDuringSchedulingIgnoredDuringExecution:
+                nodeSelectorTerms:
+                - matchExpressions:
+                  - key: game-server
+                    operator: In
+                    values:
+                    - "true"
+          tolerations:
+          - key: "game-server"
+            operator: Equal
+            value: "true"
+            effect: NoExecute
+          containers:
+            - name: middle-layer-app
+              image: ${region}-docker.pkg.dev/${project_id}/${repo_name}/${container_name}:latest
+              resources:
+                requests:
+                  memory: 256Mi
+                  cpu: 80m
+                limits:
+                  memory: 256Mi
+                  cpu: 80m
+---
```

Comment:
Can we put each K8s resource in a separate file? I found it wouldn't get applied otherwise and also for consistency with others?

Reply:
Addressed in new commit